packages/zfp/meta.yaml

```
@@ -0,0 +1,37 @@
+package:
+  name: zfp
+  version: 1.0.1
+
+source:
+  # There's no sdist for zfpy on PyPI, so we get the tarball from the GitHub release instead
+  url: https://github.com/LLNL/zfp/archive/refs/tags/1.0.1.tar.gz
+  sha256: 4984db6a55bc919831966dd17ba5e47ca7ac58668f4fd278ebd98cd2200da66f
+
+build:
+  # cflags: -fexceptions
+  # cxxflags: -fexceptions
+  # ldflags: -fexceptions
+  type: static_library
+  exports: requested
+  script: |
+    mkdir -p pyodide_build
+    cd pyodide_build
```

The name pyodide_build is a bit confusing as it sounds like pyodide-build library. So how about just build.

```suggestion
    mkdir -p build
    cd build
```

---

Makes sense to me. Addressed in 998bf8f600f548e78dc38203213ac6150056a0a6